dispatchr-ruby/app/models/request_item.rb

```diff
@@ -1,4 +1,9 @@
 class RequestItem < ApplicationRecord
   belongs_to :request, optional: true
   belongs_to :item, optional: true
+
+  validates :request_id, presence: true
+  validates :item_id, presence: false
```

Comment:
What's the point of having "validates :item_id" then if we don't even require it since presence is set to false? Curious